eldritch/test_combat.py

```diff
@@ -1978,5 +1978,108 @@ def testDeactivatesOnCombatLoss(self):
     self.assertTrue(witch.has_attribute("magical resistance", self.state, self.char))
 
 
+class BindMonsterTest(EventTest):
```

Comment:
In addition to regular testing, can I ask that you add BindMonster to a character's possessions for the encounter at the bank and the encounter at the hospital? We should verify that BindMonster is not usable in either of those combats.

Reply:
Done.